### ISSUE TITLE
Add automatically all fares and all subfleets of one specific airline if at add flight the corresponding airline is selected

### ISSUE THREAD

**Issue by SkygroupVirtual** [nabeelio/phpvms#1163]
Would it be possible to add a function to creating a flight thats called "add all fares of one specific airline" and "add all subfleet of specific airline"? So that you can add with just one click every subfleet of lets say DLH and all corresponding DLH fares. (now I have to add 15 subfleets and 8 DLH fares) Or maybe it would be possible that the when I select the airline for the flight, that all corresponding data would be loaded automatically?

**Comment by FatihKoz** [comment]
I am using something similar to what you need (for the subfleets) but i think that if this is implemented in core phpvms that should not be automatic ...

if done, it definitely should be user selectable and or optional ‘cause not every va/setup is the same and what we need may not fit other’s needs ;)

**Comment by SkygroupVirtual** [comment]
Yes, I agree - automatically may not suite everybody. Maybe if it would add those fares and you would have to deselect those you dont want at that flight. Or just make it selectable to add all fares of one airline and all subfleets of one airline to a flight with just one (or two :) ) click.

**Comment by FatihKoz** [comment]
Same logic applies to subfleets, why should i want to have a feature for adding long range fleet to a domestic short haul (just because they are using the same airline) ? 

And when it comes to selections, current system is almost doing that with more clicks  ;)

So in short, what you need is pretty much your va/setup specific ‘cause you have lots of fares and lots of subfleets and this is why i am kindly suggesting you to have your own function/method to handle your needs (no bad feelings here) ... You can have a function to add all or some subfleets of an airline to all of its flights or some of it with just one click.

Same logic applies to flight fares too. I can share more details if Tom needs it ;)
